modules/workload-identity/main.tf

```diff
@@ -40,6 +41,8 @@ resource "kubernetes_service_account" "main" {
   }
 }
 
+# TODO: add the annotation to existing service accounts automatically
```

Comment:
Do we want to resolve this TODO before merging, or remove it?

Reply:
Sorry -- I missed the WIP in the PR name. Carry on.

Reply:
It's ready now.